i18n/spanish.json

```diff
@@ -1,637 +1,628 @@
 {
-    "need-manage-messages-permission" : "No tengo permiso para `Administrar mensajes`",
-    "need-manage-roles-permission" : "No tengo permiso para `Administrar roles`",
-    "need-embed-permission" : "No tengo permiso para `Insertar enlaces`",
-    "need-attach-files-permission" : "No tengo permiso para `Insertar enlaces`",
-    "need-add-emojis" : "Me falta el permiso `agregar reacciones`.",
-    "need-connect-permission" : "No tengo permiso para `Conectarme` en esta canal de voz !",
-    "need-speak-permission" : "No tengo permiso para `Hablar` en este canal de voz !",
-    "need-embed-permission-for-goodbye" : "No tengo permiso para `Insertar enlaces` Es necesario para enviar mensajes de despedida en este canal.",
-    "need-embed-permission-for-welcome" : "No tengo permiso para `Insertar enlaces` Es necesario para enviar mensajes de bienvenida en este canal",
+    "need-manage-messages-permission" : "No tengo permiso para` Administrar mensajes` ",
+    "need-manage-roles-permission" : "No tengo permiso para` Administrar roles` ",
+    "need-embed-permission" : "No tengo permiso para 'Insertar enlaces'",
```

Comment:
You're not supposed to modify this sepcial character, please, keep using the ` instead of '